src/adcm_pytest_plugin/docker_utils.py

```diff
@@ -140,7 +140,12 @@ def get_initialized_adcm_image(
 
 def init_adcm(repo, tag, adcm_repo, adcm_tag, pull, dc=None):
     dw = DockerWrapper(dc=dc)
-    adcm = dw.run_adcm(image=adcm_repo, tag=adcm_tag, remove=False, pull=pull)
+    if dc and "localhost" not in dc.api.base_url:
```

Comment:
neither code(comments) nor commit message describes the problem.

Reply:
If docker is not on localhost. Aren't it clear?

Reply:
Added comment

Reply:
This i understood, what happes if docker is not on localhost?
`base_url[base_url.rfind("/") + 1:base_url.rfind(":")]` - without debug i cant know what will be result of this operation.

U already know what is example of base_url. Why not to leave this data in commit message or in commets near this code?

Reply:
```suggestion
    if dc and "localhost" not in dc.api.base_url:
      base_url = dc.api.base_url  # http+docker://localhost  
```
I am sure after one month of time u will need to use a debugger to understand why this code is written in this way.
Or describe this code in commit message.
I very like this article https://chris.beams.io/posts/git-commit/

Reply:
I am sorry if i mentioned wrong line in my message, but main problem is with this operation - `base_url[base_url.rfind("/") + 1:base_url.rfind(":")]`